.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,243 @@
+# Contributing
+
+👍🎉 If you’re here to contribute to this project: thanks for taking the time! 🎉👍
+
+The following is a set of guidelines for contributing to this project.
+The goal is to make things go more smoothly. If you think that's not the case, feel free to propose changes.
+
+
+## Bug reports
+
+If you find a bug or a documentation issue, please [report](https://github.com/dexidp/helm-charts/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.md) it or even better: fix it.
+
+Read on to learn more about submitting changes.
+
+
+## Feature requests
+
+If you think a feature is missing, please [report](https://github.com/dexidp/helm-charts/issues/new?assignees=&labels=kind%2Fenhancement&template=feature_request.md) it or even better: implement it.
+
+If you report it, please describe your use case concretely and with as much details as possible.
+Hiding details behind abstractions makes it harder to understand the problem.
+
+If you decide to take a stab at implementing the feature,
+read the rest of this guide to learn more about submitting changes.
+
+
+## Opening a pull request
+
+The project accepts changes in the form of pull requests (or PRs).
+Pull requests allow maintainers to properly review the patch before accepting it
+and lets the CI ([GitHub Actions](https://docs.github.com/en/actions)) run automated checks to make sure the proposed change doesn't break anything.
```

Comment:
This is the first time `CI` is used, use the full form before abbreviating it.
```suggestion
and lets the continuous integration (CI) system, in this case [GitHub Actions](https://docs.github.com/en/actions), run automated checks to make sure the proposed change doesn't break anything.
```

Do we need to refer to GitHub actions here?